lib/matplotlib/scale.py

```diff
@@ -82,6 +82,97 @@ def get_transform(self):
         return IdentityTransform()
 
 
+class FuncTransform(Transform):
+     """
+     A simple transform that takes and arbitrary function for the
+     forward and inverse transform.
+     """
+
+     input_dims = 1
+     output_dims = 1
+     is_separable = True
+     has_inverse = True
+
+     def __init__(self, forward, inverse):
```

Comment:
Is there a reason why FuncTransform takes the two functions as separate args but FuncScale takes the pair as single arg?

Reply:
Just because the kwarg gets passed in as a tuple, whereas to me it makes more sense for the transform to take them separately.  But I don't feel very strongly about it.